src/upper-level-components/Navigation.jsx

```diff
@@ -34,6 +35,10 @@ function Navigation(props) {
 
   const loginButtonConfig = {}
 
+  loginButtonConfig[Pages.google.displayName] = {
+    loginState: 'true',
```

Comment:
Just curious if this needs to be a string as opposed to the boolean `true`?

Reply:
Good point... I'll change this.

Reply:
Ah, yes, I remember why this is a string. The loginState value is used in the StatusBadge component under src/components/general/StatusBadge.jsx, where it checks against the following strings to determine state: 'idle', 'waiting', 'true', 'notSelected', 'fail', and else.